catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DuplicateCommands.java

```diff
@@ -106,32 +108,37 @@
                     + "\tComplex:   search --cql \"title like 'some text' AND modified before 2012-09-01T12:30:00Z\"")
     String cqlFilter = null;
 
-    private List<Metacard> failedMetacards = Collections
-            .synchronizedList(new ArrayList<Metacard>());
+    @Option(name = "--max", required = false, aliases = {
```

Comment:
Can we make this --maximum with an alias of -max, or maybe --maximumMetacards and -mm, or maybe --maximum and -m and change multiThreaded alias to -M.  Our usage and aliases seem quite horrible, and out of order.  I will comment on the others in another ticket.  But it seems the general convention is --<complete-name> and -<shortened-name or initial>.  However, you look at our help file and it is all over the place, including stuff like -lastmonths and -month (note no double dash and -e and -enddate, where you have no double dash and the shorter one is the name, not the alias.  It ends up looking very amateur when printing it out from command line